tests/stubtest_allowlists/py310.txt

```diff
@@ -40,6 +42,8 @@ functools.partialmethod.__get__
 functools.singledispatchmethod.__call__
 gettext.install
 gettext.translation
+hashlib.sha3_\d+
+hashlib.shake_\d+
```

Comment:
Why do these need to be allowlisted?

Reply:
In Python 3.9, hashlib will use hashes from OpenSSL when available. This means that they look like built in functions, instead of classes

```
>>> hashlib.sha3_224
<class '_sha3.sha3_224'>
```
to
```
>>> hashlib.sha3_224
<built-in function openssl_sha3_224>
```

Reply:
Might be good to add a comment about this.